src/app/components/no-page/no-page.component.ts

```diff
@@ -0,0 +1,16 @@
+import { Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+
+@Component({
+  selector: 'app-no-page',
+  templateUrl: './no-page.component.html',
+  styleUrls: ['./no-page.component.scss']
+})
+export class NoPageComponent implements OnInit {
+
+  constructor() { }
+
+  ngOnInit(): void {
```

Comment:
It's better to get rid of unused methods and constructors to keep code more clean.

Reply:
Deleted